/**
 * Ports for each of the services defined for OpenCV.
 * These will be rendered obsolete by an implementation of a multiplexing
 * client for all client languages.
 */

const int64 basePort = 9090

const int64 matUtilPortOffset = 0
const int64 features2DPortOffset = 1
